### ISSUE TITLE
false positive `Commas should never have a space before` lint for `switch` statements

### ISSUE THREAD

**Issue by IndrajeetPatil** [jimhester/lintr#708]
The following code

```r
  switch(
    method,
    "One Sample t-test" = ,
    "Two Sample t-test" = ,
    "Paired t-test" = "t-test"
)
```

produces a (in my opinion) false positive lint:

> Commas should never have a space before

Note that I can just remove the white space before `,`, but then there would be a lint about expected whitespace around `=` operator.

**Comment by AshesITR** [comment]
Hi @IndrajeetPatil!
This is a duplicate of #540 IINM.

**Comment by AshesITR** [comment]
Can you please test with the current dev version?
This particular instance of the issue should be fixed by #499.

**Comment by IndrajeetPatil** [comment]
Thanks! Works for me.
